test/functional/feature_taproot.py

```diff
@@ -1201,7 +1201,7 @@ def set_test_params(self):
         self.num_nodes = 2
         self.setup_clean_chain = True
         # Node 0 has Taproot inactive, Node 1 active.
-        self.extra_args = [["-par=1", "-vbparams=taproot:1:1"], ["-par=1"]]
+        self.extra_args = [["-par=1", "-vbparams=taproot:@1:@1"], ["-par=1"]]
```

Comment:
While you're at it, the comment about which node is active is the wrong way around.

Reply:
No? Node 0 has taproot inactive and setting the vbparams in this way (startheight=1, timeoutheight=1) ensures that it cannot activate on regtest.

Reply:
No, taproot is always active on regtest: https://github.com/bitcoin/bitcoin/blob/968bc1e07b4a8a359876f5b8035fb4c369d04bad/src/chainparams.cpp#L418

This had me massively confused too, but you can tell from `testmempoolaccept` if you manually craft a (broken) taproot transaction: https://github.com/BitMEXResearch/forkmonitor/blob/master/spec/models/chaintip_spec.rb#L377 (`mempool_broken_tap` is created by node A, which has no extra_args: https://github.com/BitMEXResearch/forkmonitor/blob/master/spec/models/chaintip_spec.rb#L22)

Reply:
Ah crap, was reading the brackets wrong. Nvm.